lib/bokken_web/controllers/organizer_controller.ex

```diff
@@ -5,13 +5,14 @@ defmodule BokkenWeb.OrganizerController do
   alias Bokken.Accounts.Organizer
 
   action_fallback BokkenWeb.FallbackController
+  defguard is_organizer(conn) when conn.assigns.current_user.role === :organizer
 
-  def index(conn, _params) do
+  def index(conn, _params) when is_organizer(conn) do
```

Comment:
I don't think it makes sense to block the index and show routes. Those should be accessible to everyone in the platform. What do you think @RuiL1904 ?